provision-proctor/setup.sh

```diff
@@ -174,5 +191,11 @@ bash ./build_deploy_sentinel.sh -r $resourceGroupProctor -g $registryName -n ${t
 echo "6-Build and deploy leaderboard website to AKS  (bash ./build_deploy_web.sh -m $proctorName -d <dnsURL>)"
 bash ./build_deploy_web.sh -m $proctorName -d $dnsURL
 
-echo "7-Clean the working environment"
+echo "7-Provision CosmosDB"
```

Comment:
We should have this and Azure Functions before so we can pass the value into Sentinel and Leaderboard.